### PR TITLE
Make link open message OS-agnostic

### PR DESCRIPTION
**Pull request by k0nserv**
Make link instructions in onboarding OS aware

### CODE REVIEW

File: lib/danger/commands/init.rb [hunk 1]
```diff
@@ -244,7 +244,7 @@ def unsure_token
     end
 
     def note_about_clicking_links
-      ui.say "Note: Holding cmd ( ⌘ ) and #{ENV['ITERM_SESSION_ID'] ? '' : 'double '}clicking a link will open it in your browser."
```
Comment (segiddins) [inline lib/danger/commands/init.rb]: Couldn't this infer whether it's running on OS X?

Comment (k0nserv) [inline lib/danger/commands/init.rb]: Yeah that's a really good point

Comment (k0nserv) [inline lib/danger/commands/init.rb]: Know a reliable way to do that?

Comment (k0nserv) [inline lib/danger/commands/init.rb]:
```ruby
Gem::Platform.local.os == "darwin"
```

Seems reasonable

Comment (k0nserv) [inline lib/danger/commands/init.rb]: Is `Gem::Platform` always available?

Comment (orta) [inline lib/danger/commands/init.rb]: Looks like it - http://ruby-doc.org/stdlib-1.9.2/libdoc/rubygems/rdoc/Gem/Platform.html

Comment (k0nserv) [inline lib/danger/commands/init.rb]: That's what I recon too since it's in the Ruby docs. @segiddins probably knows for sure?

**Comment by segiddins** [comment]
@k0nserv yup it is

**Comment by orta** [comment]
Alright, alas this has a merge conflict, happy to merge on green ( when GH allows me to )

**Comment by k0nserv** [comment]
Rebased

**Comment by orta** [comment]
Awesome, thanks!

**Pull request merged**